### DISCUSSION TITLE
Auxillary Fan / side fan 120 x 32 only starts at 70% in fluidd

### DISCUSSION THREAD

**Discussion by winbard** [Guilouz/Creality-Helper-Script-Wiki#633] [Issues]
Hello,

it is about the right side fan 120 x 32. Fan2
When I start it in fluidd it start only at 70% very slow with loud humming noise. Under 70% it make only humming noise but not run.
At 82% it run middle without humming noise. With 100% fast without this humming noise. Normal noise.
Have someone else this too?

**Comment by dthreee** [comment]
Yes, my fan does the same with those same percentages. I thought the math in the M106 macro was causing it, but I get the same results when i bypass it and use the command SET_PIN. I have to use SET_PIN to 211 or higher for it to work at all, and it seems to hit max speed at 223.

Strange thing is the sliders on the touch screen don't correspond to either the actual fan speed or the SET_PIN value sent from Klipper. The values below 82% in the start print sequence are also causing the side fan to run under speed.

BTW, I'm still on firmware 1.3.3.5

**Reply by winbard** [reply]
Hi,
thanks for the answer.
